pdc_client/__init__.py

```diff
@@ -203,10 +205,21 @@ def get_paged(self, res, **kwargs):
         """
         if self.page_size is not None:
             kwargs['page_size'] = self.page_size
-
+            page_size = self.page_size
             if self.page_size <= 0:
                 # If page_size <= 0, pagination will be disable.
                 return res(**kwargs)
+        else:
+            page_size = 20
+        if self.page is not None:
+            allinfo = res(page=self.page, **kwargs)
+            all_count = int(allinfo["count"])
+            if all_count % page_size > 0:
+                total = (all_count / page_size) + 1
+            else:
+                total = all_count / page_size
+            print("Total %d pages, here is the %d page, the count is %s." % (total, self.page, all_count))
```

Comment:
If user requests JSON format and wants to process it further, the printed plain text needs to be skipped.

This could be printed in the function which prints out the table, though I'm still not sure if it's good idea.